webp_backport/src/main/java/de/marcreichelt/webp_backport/WebPBackport.java

```diff
@@ -38,17 +38,19 @@ static boolean isIsWebpSupportedNatively() {
     }
 
     /**
-     * Decodes a WebP image.
+     * Decodes a WebP image. This is done either by using the Android internal functionality or by using the
+     * WebPBackport library. The WebPBackport library is used only when the current Android system does not support the
+     * decoding by itself and the given encoded image has the necessary WebP file signature in the beginning of its
+     * content.
```

Comment:
Not correct yet: This method now decodes *all* image formats that Android supports: JPG, PNG, ... (including WebP).
The only difference is that it uses the native library for WebP images when needed.